packages/shared/tailwind.config.ts

```diff
@@ -125,6 +125,14 @@ export default {
         'Segoe UI Emoji',
         'Segoe UI Symbol',
       ],
+      serif: [
```

Comment:
I think in the design they also changed sans font family names, but wasn't sure if it was intented or not?

Reply:
@rebelchris comments are in scope too, asked to @tsahimatsliah yesterday

Reply:
No I mean for sans fonts they changed from something to Helvetica I think?